src/pages/overview/document-generation-api/templatetags.md

```diff
@@ -11,7 +11,8 @@ the input data to generate the output document.
 
 <InlineAlert slots="text"/>
 
-All the tags are [JSONata compliant](https://docs.jsonata.org/overview.html).
+All the tags are [JSONata compliant](https://docs.jsonata.org/overview.html).<br/>
+All tags should be kept within a 1000-character limit.
```

Comment:
Shall we also mention about limit of tag value?

Reply:
not mentioning it as it is limited to regex functions use case.